torch/_inductor/codegen/wrapper.py

```diff
@@ -307,6 +307,7 @@ def __init__(self):
         self.supports_intermediate_hooks = True
         self.expr_printer = pexpr
         self.cached_thread_locals = set()
+        self.user_defined_kernel_count = 0
```

Comment:
I could make this unique to each kernel name, i.e. Dict[str, int] but other than nicer function names, it doesn't really help